src/components/connection_handler/src/connection_handler_impl.cc

```diff
@@ -1040,6 +1042,20 @@ bool ConnectionHandlerImpl::CanStartProtectedService(
   return true;
 }
 
+uint32_t ConnectionHandlerImpl::FindAppIdBySession(
+    const transport_manager::ConnectionUID connection_handle,
+    const uint8_t& session_id) const {
+  const uint32_t conn_key = KeyFromPair(connection_handle, session_id);
+  std::list<int32_t> sessions_list;
+  uint32_t device_id, app_id;
+
+  if (GetDataOnSessionKey(conn_key, &app_id, &sessions_list, &device_id)) {
```

Comment:
@tkireva Here we can skip session_list and device_id by replacing them with NULLs in the function call. Also it would be good to initialize app_id.